oci/oci.go

```diff
@@ -514,20 +514,34 @@ func (r *Runtime) StopContainer(c *Container, timeout int64) error {
 		timeout = 10
 	}
 	done := make(chan struct{})
+	// we could potentially re-use "done" channel to exit the loop on timeout
+	// but we use another channel "chControl" so that we won't never incur in the
+	// case the "done" channel is closed in the "default" select case and we also
+	// reach the timeout in the select below. If that happens we could raise
+	// a panic closing a closed channel so better be safe and use another new
+	// channel just to control the loop.
+	chControl := make(chan struct{})
```

Comment:
nit comment.  I'm finding this first sentence to be very confusing.  Perhaps:

we could potentially re-use the "done" channel to exit the loop on timeout but we use another channel "chControl" to prevent hitting the case where the "done" channel is closed in the "default" select case at the same time that we reach the timeout in the select below.